content/en/events/kcseu/faq.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Frequently Asked Questions (FAQ)"
+type: docs
+weight: 4
+---
+
+- [When is the summit taking place?](#when-is-the-summit-taking-place)
+- [Where is the summit located?](#where-is-the-summit-located)
+- [Will I be able to attend remotely?](#will-i-be-able-to-attend-remotely)
+- [Why is vaccination required to attend in-person?](#why-is-vaccination-required-to-attend-in-person)
+- [Why do I need to be a Kubernetes Org member to attend in-person?](#why-do-i-need-to-be-a-kubernetes-org-member-to-attend-in-person)
+- [What is a sponsored attendee?](#what-is-a-sponsored-attendee)
+- [I am not an Org member. Can I still attend?](#i-am-not-an-org-member-can-i-still-attend)
+- [Will there be a New Contributor Workshop?](#will-there-be-a-new-contributor-workshop)
+- [I have a question! How do I contact the event staff?](#i-have-a-question-how-do-i-contact-the-event-staff)
+
+### When is the summit taking place?
+
+May 16th, the Monday of [KubeCon Europe].
+
+[KubeCon Europe]: https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/
```

Comment:
can we move these links to the end of the doc, per style guidelines?

Reply:
I leave them in their "logical area" unless they are used more than once in which case they goto the bottom.